src/cairoWriter/writers/variableDeclarationStatementWriter.ts

```diff
@@ -27,6 +29,29 @@ export class VariableDeclarationStatementWriter extends CairoASTNodeWriter {
     const documentation = getDocumentation(node.documentation, writer);
     const initialValueType = safeGetNodeType(node.vInitialValue, this.ast.inference);
 
+    const assertUnc = `assert(overflow == false, 'Overflow in unchecked block');`;
+    let isUnc256 = false;
+    let isUnc = false;
+    if (node.vInitialValue instanceof FunctionCall) {
+      if ((node.vInitialValue as FunctionCall).vExpression instanceof Identifier) {
+        const funcName = writer.write(node.vInitialValue);
+        if (funcName.includes('_overflow_') || funcName.includes('_overflowing_')) {
```

Comment:
Where `_overflow_` functions come from? If it's from warplib let's unify the naming so only one condition needs to be checked

Reply:
is bacause the overflow subtraction function for 256 have just "overflow" instead of "overflowing"

Reply:
I see, I'll reach corelib authors on that

Reply:
I've wanted to wait with the PR until it gets fixed in the corelib, but it didn't happen so we'll merge it regardless and it'll be fixed later